### PR TITLE
Typing Speed Test Website

### PR DESCRIPTION
**Pull request by khushi-purwar**
#157  Added Typing Speed Test Website. Please review my PR @chicken-biryani 

ss1

### CODE REVIEW

**Review by harshita2216 — CHANGES_REQUESTED** [review]
Great work @khushi-purwar. Just add it to index.md and then we shall be good to go! 🚀

File: Typing Speed Test Website/style.css [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+.mainBtn {
+  border-radius: 20px;
+  padding: 10px 20px;
+  font-size: 1rem;
+  background: #ecf0f1;
+  border: 5px solid #2980b9;
+  cursor: pointer;
+}
+
+
```
Comment (chicken-biryani) [inline Typing Speed Test Website/style.css:57]: Remove the extra line.

File: Typing Speed Test Website/script.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+		playGame();
+	} else if (this.innerText == "Done") {
+		typedWords.disabled = true;
+		btn.innerText = "Start";
+		endGame();
+	}
+})
+
+
+
```
Comment (chicken-biryani) [inline Typing Speed Test Website/script.js:63]: Remove the extra lines

File: Typing Speed Test Website/index.html [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+			<textarea name="" id="mywords" cols="100" rows="10" placeholder="Click on the Start Button and Type Above Appearing Text as it is..."></textarea>
+			<button id="btn" class="mainBtn">Start</button>
+		</div>
+	</div>
+	<script src="script.js"></script>
+</body>
+
+</html>
+
+
```
Comment (chicken-biryani) [inline Typing Speed Test Website/index.html:26]: Remove the extra line

**Comment by khushi-purwar** [comment]
> Great work @khushi-purwar. Just add it to index.md and then we shall be good to go! 🚀

Added in the index.md

**Pull request merged**